data/helpers.d/network

```diff
@@ -47,7 +47,7 @@ ynh_find_port () {
 # | arg: path - the web path to check the availability of
 ynh_webpath_available () {
 	local domain=$1
-	local path=$(ynh_normalize_url_path $2)
+	local path=$2
```

Comment:
>But, this path should be fixed immediately, that is why I think we should use this helper only at the beginning of the install script.

Okay, agreed on that. If you like, we can remove `ynh_normalize_url_path` from both helpers so that everything is consistent ;)